.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - checkout
       - setup_remote_docker
 
-      - run: go get github.com/golang/dep/cmd/dep
+      - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Not ~~sure if this is~~ the best approach -- although it's the simplest.

Alternative would require a lot more lines of code (we need to retrieve the latest release tag by ourselves, download this release from GitHub, set the right permissions and put it in to the right place -- basically the same as what the `install.sh` is doing for us).